ni_measurementlink_service/_internal/parameter/serializer.py

```diff
@@ -224,11 +227,18 @@ def _deserialize_enum_parameters(
         parameter_metadata = parameter_metadata_dict[id]
         if get_enum_values_annotation(parameter_metadata):
             enum_type = _get_enum_type(parameter_metadata)
+            is_protobuf_enum = enum_type is int
```

Comment:
Please test `IntEnum` as well. An `IntEnum` is an instance of both `Enum` and `int`.

Reply:
Added tests for IntEnum.